mantidproject/mantid
# macOS bundle matplotlib

This bundles the system `matplotlib` with the package, so that its version is consistent with the `mpl_toolkits` from PR #21913 .

**To test:**
1. Download and install the package from build artefacts.

2. Run MantidPlot, make sure there is no error on start up and on `Interfaces->Direct->DGSPlanner`.

3. Bring in the new `matplotlib` also  
```
sudo pip install "matplotlib>=2.1.2"
```
Repeat 2 and make sure there is no problem still.



No issue, no note. 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
